CHANGELOG.md

```diff
@@ -2,6 +2,57 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [2.31.0](https://github.com/serverless/serverless/compare/v2.30.3...v2.31.0) (2021-03-23)
```

Comment:
I was not 100% sure about keeping all of the entries in Bug Fixes & Maintenance Improvements - please let me know if something else should be removed as well